All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+
+**Jaye Harrill**
+Is there a playbook as to what the checks are to do when that happens, all the checks that need to be need to be done?
+
+**James Hancock**
+Not that I'm aware of.
+
+**Pooja Ranjan**
+So for that, we have created this emergency comm process document, sometimes back during the Constantinople post mortem only. So it suggests that we should be having a list of key audiences, those are going to be, you know, emergency comms members. And we are trying to create this list. Generally, we use this kind of list at the time of upgrade or hard fork to communicate with our key stakeholders to let them know what's going on if something urgent comes up. So that's it, and the other thing was we came up with a list of preferred channels of communication, because it's important that we should be reaching people, more people and as quickly as possible. So these are the preferred, appropriate channels that we suggested we should be using.
+
+**Jaye Harrill**
+That make sense. I'm just quickly looking at this. Certainly understanding the list of who to contact, this is public, so I know that it would have to be a trusted group of people to do that. But I feel like if there's a little bit more direction in this document that that team could be formed. I don't know. It's honestly more of an operations role with someone who can understand how to form something that doesn't provoke worry, but provokes calm, immediate action of the right things rather than a reactionary thing. And so I don't think that's very hard. There's some talented comms people in the space.
+
+**Pooja Ranjan**
+Yeah, so if you could put me in touch with them, or maybe if you feel like you could be helpful to add more to this document, but as you mentioned, as information can be sensitive, and we would want to keep it in a closed group, we could work together on that.
+
+**Jaye Harrill**
+Yeah, I'd be happy to try and find some people. I'd be happy to be on the team, too. I'm just - there is a process that already exists, and there is already a list of people to contact, so I don't want to change that. I think it's more or less here. All it's really needed is the people to be able to execute on the thing when it's needed and to have the professionalism to work together and say the right things. And I think that could be honestly coordinated through a little Telegram chat that's limited in how many people are in it.
+
+**Pooja Ranjan**
+So the executors here are basically the Cat Herders itself. We reac out to the group of people in the Cat Herders, and we reach out to these people. And James is the hard fork coordinator, so we work together. And some people are reached by James, and that's how we have been managing it so far. But yeah, happy to have you on board, too.
```

Comment:
o the executors here are basically the Cat Herders itself. We reach out to the group of people on the list and James is the hard fork coordinator, so we work together. Some people are reached by James, and that's how we have been managing it so far. But yeah, happy to have you onboard, too.